app/middleware.go

```diff
@@ -183,3 +194,48 @@ func (nr *NewRelicMiddleware) Serve(next echo.HandlerFunc) echo.HandlerFunc {
 		return nil
 	}
 }
+
+const metricName = "response_time_milliseconds"
+
+// ResponseTimeMetricsMiddleware struct encapsulating DDStatsD
+type ResponseTimeMetricsMiddleware struct {
+	DDStatsD *middleware.DogStatsD
+}
+
+//ResponseTimeMetricsMiddleware is a middleware to measure the response time
+//of a route and send it do StatsD
+func (responseTimeMiddleware ResponseTimeMetricsMiddleware) Serve(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+
+		metricTagsMap := make(map[string]interface{})
+		c.Set("metricTagsMap", metricTagsMap)
```

Comment:
You only need to set the "metricTagsMap" in the first middleware of the middleware chain. Since you have set in "LoggerMiddleware" and it is the first middleware of the chain, you don't need to do it here again.